community.md

```diff
@@ -24,17 +24,18 @@ title: Trino Community
         </div>
         <div class="slack-stats-container">
           <div class="community-stats">
-            <span>8454</span>
+            <span>11300+</span>
             Slack members
           </div>
           <div class="community-stats">
-            <span>778</span>
-            Weekly active members
```

Comment:
This too much of a fluctuating number

Reply:
To me this one would be an incredible chart or daily update as to me it's tracking the activity of the engagement of the community. I do see your point of it being useless if it's not updated with a cadence.

Reply:
Yeah .. for analysis this is a good number . but not so much for marketing..  I think there are charts about this in orbit